src/AvPortalResource.php

```diff
@@ -42,6 +42,26 @@ public function getRef(): string {
     return $this->data['ref'];
   }
 
+  /**
+   * Get the reference.
+   *
+   * @return string
+   *   The reference.
+   */
+  public function getType(): string {
+    return $this->data['type'];
+  }
+
+  /**
+   * Get the Photo URI (for PHOTOS).
+   *
+   * @return string
+   *   The photo Uri
+   */
+  public function getPhotoUri(): string {
+    return $this->data['media_json']['HIGH']['PATH'];
```

Comment:
I'm ok with this for now but shouldn't we give users the option to chose what source to use? I'm assuming there is also MED and LOW resolution pictures, and it might be needed in future use cases.